akka-bench-jmh-dev/src/main/scala/akka/stream/InterpreterBenchmark.scala

```diff
@@ -54,7 +54,10 @@ class InterpreterBenchmark {
     val lock = new Lock()
     lock.acquire()
     val sink = OneBoundedDataSink(data100k.size, lock)
-    val ops = Vector.fill(numberOfIds)(MapStage(identity[Int], Supervision.stoppingDecider))
+    val ops = Vector.fill(numberOfIds)(new PushPullStage[Int, Int] {
```

Comment:
having a PushPullStage instead of a PushStage disables the jumpBack table which speeds up things in this case, but not in others. I wanted to see the difference between the two interpreters when this optimization is not applicable.